crmprtd/__init__.py

```diff
@@ -51,3 +55,26 @@ def f_retry(*args, **kwargs):
             return f(*args, **kwargs)
         return f_retry  # true decorator
     return deco_retry
+
+
+def setup_logging(log_conf, log, error_email, log_level, name):
+    if log_conf:
+        with open(log_conf, 'rb') as f:
+            log_c = yaml.load(f)
+    else:
+        log_c = yaml.load(resource_stream('crmprtd', '/data/logging.yaml'))
+
+    if log:
+        log_c['handlers']['file']['filename'] = log
+    else:
+        log = log_c['handlers']['file']['filename']
```

Comment:
I know that this is what the code used to do, but is there any value to this assignment? We just throw out the `log` variable a few lines later. Let's axe the `else` branch of this conditional.